config/routes.rb

```diff
@@ -6,12 +6,16 @@
   resources :posts
   resources :comments, only: [:create]
   resources :likes
+  resources :friendships, only: [:create, :destroy, :index]
```

Comment:
- [ ] If we are here requiring the route for the destroy action, aligned with [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete), why are we later doing `  post '/deleterequest', to: 'friendships#delete_request'` for the same purpose?